src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -184,6 +184,9 @@ public class HelixWatch3DViewModel : DefaultWatch3DViewModel
         private double nearPlaneDistanceFactor = 0.001;
         internal const double DefaultNearClipDistance = 0.1f;
         internal const double DefaultFarClipDistance = 100000;
+        private const int DepthBiasVertexColors = 10;
+        private const int DepthBiasNormalMesh = 100;
+        private const int DepthBiasSelectedMesh = 0;
```

Comment:
Just curious how you ended up with these numbers?  Maybe worth a comment?

Reply:
Are they the same as the old shader implementation?

Reply:
I didn't reference the old shader, but I can look - I vaguely remember both depth bias and nudging were used.

The docs for depth-bias describe that the numbers can be interpreted multiple ways depending on the format of the depth buffer - I tried to find values that would work in either case. See the link above.

Reply:
to me - it looks like helix uses a 32bit depth buffer (and 8 bit stencil buffer) - 

DXGI_FORMAT_D32_FLOAT_S8X24_UINT | A 32-bit floating-point component, and two unsigned-integer components (with an additional 32 bits). This format supports 32-bit depth, 8-bit stencil, and 24 bits are unused.⁵
-- | --

which means the value is scaled by `1/(2^32)`, something like:
Example: DepthBias = 100 ==> Actual DepthBias = 100/2^32 = .00000003

very small changes in depth.

Reply:
Could you explain the difference between `VertexColors` and `NormalMesh` and why they require different depth biases?

Reply:
@aparajit-pratap - imagine the case where you have two spheres at the same position, with the same radius. One was created by the `ByGeometryColor` node -  IMO the preferred behavior is that the colored sphere appears on top of the non colored sphere.

Today this doesn't work because most DynamoModelGeometry3d objects have vertex colors - I'm not sure why. When we address that this will ensure that objects with vertex colors are drawn on top of geometry without colors)